server/templates/browser/browser2_entity.html

```diff
@@ -0,0 +1,42 @@
+{#
+    Copyright 2020 Google LLC
+   
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+   
+         http://www.apache.org/licenses/LICENSE-2.0
+   
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+   #}
+   
+   {% extends 'base.html' %}
+   
+   {% set main_id = 'dc-browser2' %}
+   {% set title = node_name + '- Graph Browser' %}
+   {% set subpage_title = 'Graph Browser' %}
+   {% set subpage_url = url_for('browser.kg_main') %}
+   
+   {% block head %}
+     <link rel="stylesheet" href={{url_for('static', filename='css/kg.min.css', t=config['GAE_VERSION'])}} >
+   {% endblock %}
+   
+   {% block content %}
+   <div class="container">
+       <div id="node" data-dcid="{{dcid}}">
+           <div class="node-about" id="node-name" data-nn="{{node_name}}">
+               {{ 'About: ' + node_name }}
+           </div>
+           <div id="node-content"></div>
```

Comment:
a loading spinner or message would be nice here

Reply:
done. I put in a simple loading message and added a todo for a better loading indicator